### PR TITLE
Sils/ged

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Comment by sils** [comment]
CC @AbdealiJK for review, didn't manage to try it out yet

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-coala==0.3.0
+coala~=0.6.0
```
Comment (AbdealiLoKo) [inline requirements.txt]: I normally use `>=` for coala version.

Btw, `>=0.5` would be better as it does work with 0.5

Note: This file isnt really used anywhere in the plugin

**Comment by AbdealiLoKo** [comment]
Both 0c44603 and 2aaa7b1 look good to me. But testing on a computer is needed as we don't have CI

**Comment by AbdealiLoKo** [comment]
Sorry. unack 0c44603d87913779296103aa09a6837b631315c1

**Comment by sils** [comment]
@AbdealiJK how's this?

**Comment by AbdealiLoKo** [comment]
ack 6599789

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
btw. it's tested by @sims1253 and works fine

**Pull request merged**